CHANGELOG.md

```diff
@@ -3,11 +3,14 @@
 
 All notable changes to this project will be documented here.
 
+### v2.1.14
+- Fix: Reverts [PR-243](https://github.com/stakater/application/pull/243), This can already be configured via `paths` [PR-247](https://github.com/stakater/application/pull/247)
```

Comment:
```suggestion
- Fix: Reverts [PR-240](https://github.com/stakater/application/pull/240), This can already be configured via `paths` [PR-247](https://github.com/stakater/application/pull/247)
```